modules/apps/27-interchain-accounts/controller/types/keys.go

```diff
@@ -0,0 +1,9 @@
+package types
+
+const (
+	// ModuleName defines the interchain accounts controller module name
+	ModuleName = "icacontroller"
```

Comment:
Here a new `ModuleName` is defined for the `controller` package. I'm open to making this `SubmoduleName` if anyone thinks it suits better, but I think this is fine. 

This is also used below for `StoreKey` as it is in other ibc and sdk modules. This is necessary to namespace the stores. Otherwise if both submodule stores were namespaced under the root module name, then an exported genesis calling into `GetAllPorts` would return the default host port - "ibcaccount" ("interchainaccount" when updated), and when a chain attempts to restart it would panic due to the capabilities with "port already bound".
It's important to note this wouldn't happen if a chain started with DefaultGenesis values, the issue would only occur with a custom/exported genesis file.